frontend/src/components/pages/teacher/TeacherRouting.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { Redirect, Route, Switch } from "react-router-dom";
+import { Box, VStack } from "@chakra-ui/react";
+
+import * as Routes from "../../../constants/Routes";
+import Page from "../../../types/PageTypes";
+import PrivateRoute from "../../auth/PrivateRoute";
+import Navbar from "../../common/Navbar";
+import NotFound from "../NotFound";
+
+import ClassroomsPage from "./ClassroomsPage";
+import DistributeAssessmentPage from "./DistributeAssessmentPage";
+import TeacherDashboardPage from "./TeacherDashboardPage";
+
+const pages: Page[] = [
+  { title: "Dashboard", url: Routes.TEACHER_DASHBOARD_PAGE },
+  { title: "Assessments", url: Routes.DISTRIBUTE_ASSESSMENT_PAGE },
+  { title: "Classrooms", url: Routes.CLASSROOMS_PAGE },
+];
+
+const TeacherRouting = (): React.ReactElement => {
+  return (
+    <Switch>
+      <VStack align="left" flex="1">
+        <Navbar pages={pages} />
+        <Box padding="1.5em 2em 0em 2em">
+          <Switch>
+            <PrivateRoute
+              component={TeacherDashboardPage}
+              exact
+              path={Routes.TEACHER_DASHBOARD_PAGE}
+              roles={["Teacher"]}
+            />
+            <PrivateRoute
+              component={DistributeAssessmentPage}
+              exact
+              path={Routes.DISTRIBUTE_ASSESSMENT_PAGE}
+              roles={["Teacher"]}
+            />
+            <PrivateRoute
+              component={ClassroomsPage}
+              exact
+              path={Routes.CLASSROOMS_PAGE}
+              roles={["Teacher"]}
+            />
+            <Redirect
+              exact
+              from={Routes.TEACHER_LANDING_PAGE}
+              to={Routes.TEACHER_DASHBOARD_PAGE}
+            />
+            <Route component={NotFound} exact path="*" />
```

Comment:
side note - did we end up deciding to keep the nav bar for NotFound? if so can we create ticket to fix this width thing
<img width="1435" alt="image" src="https://user-images.githubusercontent.com/69697180/236736580-b7f3c764-e6c9-462a-b694-468d28fc4041.png">

Reply:
wait i think this is new, i haven't seen this before. made a ticket for the next sprint.